### PR TITLE
Fix broken link in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by intrigus-lgtm**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ When working on the docs, please keep the following guidelines in mind:
   * Don't: fuzzing-network-service.md
 * Use a maximum of 80 characters per line to make reading in a console easier.
 * Make all pull requests against `dev`, see
-  [#how-to-submit-a-pull-request-to-afl](#how-to-submit-a-pull-request-to-afl).
+  [#how-to-submit-a-pull-request-to-afl](#how-to-submit-a-pull-request).
```
Comment (domenukk) [inline CONTRIBUTING.md]: We should also change the text, right?

Comment (vanhauser-thc) [reply CONTRIBUTING.md]: @intrigus-lgtm ?

Comment (intrigus-lgtm) [inline CONTRIBUTING.md]:
```suggestion
  [#how-to-submit-a-pull-request](#how-to-submit-a-pull-request).
```

Comment (intrigus-lgtm) [reply CONTRIBUTING.md]: Done.

**Comment by domenukk** [comment]
Thanks!

**Pull request merged**